recipes/source.rb

```diff
@@ -53,9 +57,11 @@
 # reduce line-noise-eyness
 execute "Extracting and Building Git #{node['git']['version']} from Source" do
   cwd Chef::Config['file_cache_path']
+  additional_make_params = ""
```

Comment:
If the condition is false, it defines var and the var can be used further. It's fine.
However, if the condition is true, and the var isn't defined, a += "something" fails:
irb(main):001:0> additional_make_params += "USE_LIBPCRE=1" if false
=> nil
irb(main):002:0> additional_make_params += "USE_LIBPCRE=1" if true
NoMethodError: undefined method `+' for nil:NilClass
	from (irb):2
	from /usr/bin/irb:12:in `<main>'
irb(main):003:0>

In this particular case, we can just use assignment (a = "val") instead of concatenation (a += "val"), but my idea is, if someone wants to add more parameters he will just add another one concatenation line with condition. Like:
additional_make_params = ""
additional_make_params += "USE_LIBPCRE=1" if node['git']['use_pcre']
additional_make_params += "USE_FEATURE1=1" if node['git']['use_feature1']
additional_make_params += "USE_FEATURE2=1" if node['git']['use_feature2']
...